### DISCUSSION TITLE
Smokeping and rrdcached

### DISCUSSION THREAD

**Discussion by mmargi** [oetiker/SmokePing#298] [Q&A]
Hello, I tried to use smokeping for about 50.000 devices in our network, but i have problem with write to rrd files.
I found that rrdcached can help - probably - but i found only one mention about support in smokeping - for BSD here https://openports.pl/path/net/smokeping. But I was not able to find solution for Debian. Know anybody how to do it, if is it possible?  
Many thanks, Margi.

**Comment by oetiker** [comment]
Are you sure you have sufficient memory on the machine where the rrds are stored ? If you see excessive read activity on your drive, then you have too little memory!

**Reply by mmargi** [reply]
Many thanks for reply. There is 32G, memory is ok, i use about 20 concurent fping probes. Now it is working relative fine, storage is SSD but I'm worried about ssd endurance when I see write rate :). Or if is any better solution how to do what I try to do? :)
